radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderArduino_RC522_LF.js

```diff
@@ -3,6 +3,22 @@ var bluetooth = require('./CommBluetooth');
 var resources = require('./Resources');
 var ReaderArduino_RC522_LF = {};
 
+/**
+ * Provides a parser to parse the RC522 LF Bluetooth device. This calls
+ * the successful every time a "frame" has been built, the error callback 
+ * if it failed to connect to the device, and the reader object sent in from
+ * RadLib.js. On each successful callback, an object with the following properties
+ * is returned:
+ *       id id of the read tag

```

Comment:
It might be helpful to have a delimiter between the property and the definition like this:
`id: id of the read tag`